cql3/statements/create_view_statement.cc

```diff
@@ -312,7 +312,7 @@ view_ptr create_view_statement::prepare_view(data_dictionary::database db) const
                 "the corresponding data in the parent table.");
     }
 
-    auto where_clause_text = util::relations_to_where_clause(_where_clause);
+    auto where_clause_text = util::relations_to_where_clause(expr::conjunction{_where_clause});
```

Comment:
Regarding my first comment about removing TRUE when factoring the expression, let's grill this one. It will result in the user using one string to create a view, but they will be given a different string here. Sounds harmless, but maybe also undesired, I can imagine frameworks trying to validate if view creation succeeded, and failing.

Reply:
There is no K_TRUE yet, so it can't happen. But I'll remove the TRUE optimization.

Regarding the text coming back unchanged, I don't think we can guarantee it, for example whitespace will definitely be mangled.